helm-charts/common/data-prep/README.md

```diff
@@ -38,21 +44,23 @@ Then run the command `kubectl port-forward svc/data-prep 6007:6007` to expose th
 Open another terminal and run the following command to verify the service if working:
 
 ```console
-curl http://localhost:6007/v1/dataprep  \
+curl http://localhost:6007/v1/dataprep/ingest  \
```

Comment:
The dataprep api has been changed for file uploading?

Reply:
Unfortunately, yes